rero_ils/modules/acq_orders/serializers/csv.py

```diff
@@ -0,0 +1,283 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Acquisition order serialization."""
+
+import csv
+
+from flask import current_app, request, stream_with_context
+from invenio_i18n.ext import current_i18n
+from invenio_records_rest.serializers.csv import CSVSerializer, Line
+
+from rero_ils.modules.acq_accounts.api import AcqAccountsSearch
+from rero_ils.modules.acq_order_lines.api import AcqOrderLinesSearch
+from rero_ils.modules.acq_receipt_lines.api import AcqReceiptLinesSearch
+from rero_ils.modules.acq_receipts.api import AcqReceiptsSearch
+from rero_ils.modules.documents.api import DocumentsSearch
+from rero_ils.modules.vendors.api import VendorsSearch
+from rero_ils.utils import get_i18n_supported_languages
+
+role_filter = [
```

Comment:
It would be great to be retrieved form the JSONSchema.

Reply:
This is a list of only "creator" roles. In the JSONSchema, we have roles that we don't need.
I rename the attribute to be more explicit